coding-guides/making-your-own-starboard.md

```diff
@@ -135,15 +135,15 @@ module.exports = class {
     if (message.author.id === user.id) return message.channel.send(`${user}, you cannot star your own messages.`);
     if (message.author.bot) return message.channel.send(`${user}, you cannot star bot messages.`);
     const { starboardChannel } = this.client.settings.get(message.guild.id);
-    const starChannel = message.guild.channels.find(channel => channel.name === starboardChannel)
+    const starChannel = message.guild.channels.cache.find(channel => channel.name === starboardChannel)
     if (!starChannel) return message.channel.send(`It appears that you do not have a \`${starboardChannel}\` channel.`); 
     const fetchedMessages = await starChannel.fetchMessages({ limit: 100 });
     const stars = fetchedMessages.find(m => m.embeds[0].footer.text.startsWith('⭐') && m.embeds[0].footer.text.endsWith(message.id));
```

Comment:
Doesn't v12 use something like channel.messages.fetch now?

Reply:
nope, it uses message.guild.channels.cache.

Reply:
Evie was asking about the removed `fetchMessages()` method that's no longer present according to the v12 documentation.